crates/router/src/routes/dummy_connector/errors.rs

```diff
@@ -29,6 +29,9 @@ pub enum DummyConnectorErrors {
 
     #[error(error_type = ErrorType::InvalidRequestError, code = "DC_06", message = "Payment is not successful")]
     PaymentNotSuccessful,
+
+    #[error(error_type = ErrorType::ServerNotAvailable, code = "DC_07", message = "Error occurred while fetching redis connection")]
+    RedisConnectionError,
```

Comment:
```suggestion
    #[error(error_type = ErrorType::ServerNotAvailable, code = "DC_00", message = "Something went wrong")]
    DummyConnectorInternalServerError,
```

@ThisIsMani Shouldn't this be ideally `DC_00` (if we're mirroring the behavior of our router itself) and the other error codes should start from 1? Can we swap the error codes of `DummyConnectorInternalServerError` and `PaymentStoringError` (so that `PaymentStoringError` would now have error code `DC_07`)?

Reply:
Changing those codes doesn't effect anything except the error messages. So we can change the codes.

Reply:
This has been fixed.

Reply:
@ThisIsMani Thanks for the fix.